app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -591,6 +591,9 @@ def clone_vm(raw_args)
         "resource_pool" => [args[:cluster], args[:resource_pool]],
         "boot_order" => [:disk],
         "annotation" => args[:annotation],
+        "virtual_tpm" => args[:virtual_tpm],
```

Comment:
Can we please update the commit message to mention that it fixes the `clone` action as well?